src/test/java/parser/ExcelParserTest.java

```diff
@@ -97,9 +98,10 @@ public void testEmptyRow() {
 
 	@Test
 	public void testCorrect() {
-		new ExcelParser("src/test/resources/test.xlsx");
+		ExcelParser ex = new ExcelParser("src/test/resources/test.xlsx");
 		assertThat(logger.getLoggingEvents(), is(asList(info("Starting parsing..."), info("Agent added"),
 				info("Agent added"), info("Agent added"), info("Finish parsing..."))));
+		assertTrue(ex.getFileFullPath().equals("src/test/resources/test.xlsx"));
```

Comment:
O would not only check that the logger says it’s been created but also that the content of the excelparser contains all the agents in the excel.

Reply:
Yes sure! I will implement that test in the InsertDataTest class which I will implement right away!

Reply:
Excuse me, yes, in this test file is where I will implement that, I got confused with the list of agentsToInsert.

Reply:
hahah I already thought you would figure it out 😸 .